.github/workflows/deploy-dev.yml

```diff
@@ -1,6 +1,6 @@
-name: Manuell bygg/deploy dev
+name: Manuell bygg/deploy DEV
 
-run-name: Manuell bygg/deploy dev - ${{ github.event.head_commit.message }}
+run-name: Manuell bygg/deploy til DEV - ${{ github.event.head_commit.message }}
```

Comment:
Ikke relatert til denne PR men vet ikke om den "head_commit.message" git det du forventer?
Eller hvor ser man denne informasjonen? 

<img width="828" alt="image" src="https://github.com/user-attachments/assets/08430f47-8b77-426c-9ac7-dce03cf60709">

Reply:
Forskjellige event-typer har commit-meldingen på forskjellige stier. For workflow-runs er det tydeligvis `github.event.workflow_run.head_commit.message` som gjelder.